tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 3.2.0
-envlist=py,linter
+envlist=py{310,311,312},linter
```

Comment:
I've been sticking to just 1 version of python locally to speed things up. Can we leave this just `py` and rely on CI to run the matrix?

Reply:
makes sense!